05/app.js

```diff
@@ -11,25 +11,30 @@ Student.prototype.addGrade = function (subject, grade) {
     this.grades[subject].push(grade);
 };
 
-Student.prototype.getAverageGrade = function (subject) {
+function averageCounter(item){
```

Comment:
`item` to nazwa, która nie wiele mi mówi - poza tym sugeruje, że to jeden element, a widzę, że to tablica więc może być więcej elementów ;P

PS. Tą funkcję można też zrobić jako metodę tj. przy pomocy `prototype`